### PR TITLE
Rewrite score computing, fixes #69

### LINKED ISSUE #69: Rewrite score computing
* Remove scores from DB
* Create a ScoredVotes view (vote, [representative, proposal, date], score)
* Rewrite the DossierScores view that aggregates entries from the previous one over dossiers
* Create a Scores view that aggregates entries from the previous one over representatives
* Remove score update facilities everywhere
* Update the API accordingly

Note: include hooks to set up different score aggregation rules (see #63)

### PR DESCRIPTION
**Pull request by njoyard**
This PR removes static, asynchronous representative score updates by replacing plain models with views in the `representatives_recommendations` app:

* `votescores` computes the score consequence of each vote on the representative score ; it is mapped to the new VoteScore model that replaces ScoredVote (which extended Vote and computed scores in python).
* `dossierscores` was introduced in a previous changeset but was rewritten to use `votescores`; it aggregates votes over proposals from a single dossier and returns the score contribution of each dossier on representatives scores. It is mapped to the DossierScore model.
* `representativescore` aggregates scores from the `dossierscores` view over all dossiers for each representative and gives a direct queryable access to the score of each representative. It is mapped to the RepresentativeScore model.

This PR also removes score update facilities as they now don't have any use.

TL;DR: no more manual score updates, scores are computed on the fly.

### CODE REVIEW

**Comment by jpic** [comment]
Brutal :guitar:
